playwright-e2e/authentication/auth-base.ts

```diff
@@ -40,5 +40,6 @@ export async function createAccountWithEmailAlias(
   }
   const emailAlias = generateEmailAlias(email);
   await fillInEmailPassword(page, { email: emailAlias, password });
-  return email;
+  console.log(`Created account with email: ${emailAlias}`);
```

Comment:
Is this console.log() necessary?

Reply:
It's for me, at least. I often run tests on localhost. If I know the email, I can manually log in to check.